hogben/tests/test_fisher.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pytest
+
+from hogben.simulate import simulate
+from hogben.utils import fisher
+from refnx.reflect import SLD
+from unittest.mock import patch
+
+
+@pytest.fixture
```

Comment:
I would also add a refl1d model fixture as we should make sure we are testing for both types of input

Reply:
I was actually working on this, just before I read this review. I was testing the coverage using `pytest --cov-report html hogben/tests/ --cov=hogben`, which gives very useful overviews when entire lines are missed. From which I noticed the ref1d case was missed, as well as the no data case.